### ISSUE TITLE
supervisor status show Service is running but pid is 0

### ISSUE THREAD

**Issue by SjhZju** [Supervisor/supervisor#758]
i use soft link to keep service's file and supervisord.conf. When i update my service(i use supervisor to stop service, then remove soft link and create a new soft link),when i execute update and start Service, it shows ok. But when i execute status, it shows service is running, but the pid is 0(ZERO). And i can not stop or restart it. Only after i stop all the rest service and kill Supervisor then restart, it turn out to be ok.

**Comment by mnaberez** [comment]
Duplicate of #445.
